examples/Workflows/README.md

```diff
@@ -1,3 +1,8 @@
 # Workflow examples
 
- * [Pipeline/Sequential workflow using densenet161 image classifier example](Densenet%20Image%20Classifier)
\ No newline at end of file
+The following links provide examples on how to implement workflows with different models. The workflow feature can be used for serving an ensemble of models and python functions through workflow APIs. A workflow is executed as a DAG where the nodes can be either model (MAR files) or functions specified in the workflow handler file. The DAG need not contain additional functions if not required. Typically, the function nodes are used for processing or augmenting intermediate data or aggregating data from multiple nodes. Currently, only sequential workflow is supported fully.
+For a more detailed explanation of Workflows and what is currently supported please refer to the main [documentation](../../docs/workflows.md)
```

Comment:
@maaquib regarding the supported workflow "only sequential workflow is supported fully", I wonder if we should consider that other workflows can be supported using the pre-processing node to dispatch the input to parallel models in the DAG. We need to clarify this in the main workflow doc as well.

Reply:
@HamidShojanazeri That is a good point. I think we should have a more generic way of passing data across the workflow, like a context object before calling it fully supported thought. Let me know your thoughts